lib/model/query/entities.js

```diff
@@ -245,14 +251,17 @@ const _createEntity = (dataset, entityData, submissionId, submissionDef, submiss
   return entity;
 };
 
-const _updateEntity = (dataset, entityData, submissionId, submissionDef, submissionDefId, event, forceOutOfOrderProcessing = false) => async ({ Audits, Entities }) => {
+// Extra flags
+// - forceOutOfOrderProcessing: entity was in backlog and was force-processed, which affects base version calculation
+// - createSubAsUpdate: submission was meant to *create* entity (and should be parsed as such) but is applied as an update
+const _updateEntity = (dataset, entityData, submissionId, submissionDef, submissionDefId, event, forceOutOfOrderProcessing = false, createSubAsUpdate = false) => async ({ Audits, Entities }) => {
   if (!(event.action === 'submission.create'
     || event.action === 'submission.update.version'
     || event.action === 'submission.reprocess'))
     return null;
 
   // Get client version of entity
-  const clientEntity = await Entity.fromParseEntityData(entityData, { update: true }); // validation happens here
+  const clientEntity = await Entity.fromParseEntityData(entityData, createSubAsUpdate ? { create: true } : { update: true }); // validation happens here
```

Comment:
How about adding a short test that verifies that the submission is parsed as a create even though it's applied as an update? For example, if the submission is missing the entity label, that should be an error because it would have been rejected as a create.